### DISCUSSION TITLE
Sessions: Multiple devices

### DISCUSSION THREAD

**Discussion by Nikolastual** [adonisjs/core#2038] [Help]
Hi!

Does Adonisjs offer multiple sessions or tokens to identify user devices?
this system allows secure access to the user account via several devices.
each user will be able to disconnect from all sessions.

I think this system may be possible with socket.io?

Thanks

**Comment by thetutlage** [comment]
Nope. Also I believe you are talking about the auth session and not the session in general. You have to manually track login sessions of a user inside the database, we do not do it by default

**Reply by Nikolastual** [reply]
So I can't use adonis authentication sessions for this system?

**Reply by thetutlage** [reply]
You can. Its just that tracking them inside the database is something you have to do manually
